CLAUDE.md

```
@@ -1,130 +1,296 @@
+npm run test             # Jest with coverage (enforced thresholds)
+npm run test:watch       # Jest watch mode (no coverage)
+npm run test:e2e         # Playwright E2E tests
+
+npm run validate:arch    # Architecture layer dependency validation
+```
+
+## Architecture
+
+### Layer Overview
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)